xml/net_sync.xml

```diff
@@ -80,782 +74,369 @@
    plan to use them, read the program documentation.
   </para>
 
-  <para>
-<!-- http://doccomments.provo.novell.com/comments/28800 -->
+  <!-\-<para>
+<!-\\- http://doccomments.provo.novell.com/comments/28800 -\\->
    These days file synchronization can also be done with a cloud computing
    solution.
+  </para>-\->
+ </sect1>-->
+ <warning>
+  <title>Risk of Data Loss</title>
+  <para>
+   Before you start managing your data with a synchronization system, you
+   should be well acquainted with the program used and test its functionality.
+   A backup is indispensable for important files.
```

Comment:
A backup is indispensable for important files.

**Make sure to back up your important files**